configs/apache_echarts.json

```diff
@@ -13,9 +18,6 @@
     "lvl5": ".nuxt-content h6",
     "text": ".nuxt-content p, .nuxt-content li"
   },
-  "custom_settings": {
-    "attributesForFaceting": ["language"]
-  },
```

Comment:
This can be restored with `lang` instead of `language` and it's good!

Reply:
Fixed! Thanks!